views/home/index.hbs

```diff
@@ -1,2 +1,6 @@
+<form action="/users/new">
+    <input type="submit" value="Sign Up" />
+</form>
+{{!-- <a href='users/new/'><button>NewButton</button></a --}}
```

Comment:
Need to remove this extra line. But we can still merge it and delete it in the next branch.